### DISCUSSION TITLE
MOSAIK or mosaik?

### DISCUSSION THREAD

**Discussion by bpalmintier** [OFFIS-mosaik/mosaik#21] [General]
What is the preferred capitalization for mosaik when being referred to outside of github/the website, such as in a technical paper? It seems that the website uses all lower case, "mosaik," (except when starting a sentence), but the publications, even for 3.0, use all capitals, "MOSAIK," in the title. Interestingly at least the 3.0 report switches to all lower case in the body text.

I'm asking b/c I have had reviewers flag this discrepancy and suggest that I should use all caps (MOSAIK) in the body of a paper to match the title shown in the References. But, this seems at odds with most other written works.

**Comment by eike** [comment]
The preference is to use "mosaik" everywhere. You can use "Mosaik" at the beginning of a sentence if lowercase conflicts with style guides. Using small caps is also fine. It is a bit unfortunate that the mosaik 3.0 paper chose to go with "MOSAIK", I actually don't know how that happened.
